core/src/components/input/input.ionic.scss

```
@@ -33,6 +134,21 @@
   min-height: globals.$ion-scale-1400;
 }
 
+// Input Shapes
+// --------------------------------------------------
+
+:host(.input-shape-soft) {
+  --border-radius: #{globals.$ion-border-radius-200};
```

```suggestion
  --border-radius: #{globals.$ion-soft-xl};
```

This would match Figma